api/interservice/authn/authenticate.pb.validate.go

```diff
@@ -33,6 +33,9 @@ var (
 	_ = ptypes.DynamicAny{}
 )
 
+// define the regex for a UUID once up-front
+var _authenticate_uuidPattern = regexp.MustCompile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$")
+
```

Comment:
Huh this means we must have also upgraded our deps...? Not a bad thing. It means we could use the `uuid` well-known string type in a few places, maybe.

Reply:
Yeah, installing `buf` via go modules requires pretty new versions of all the grpc/proto things, we should be completely up to date now 🎉